itl/plugins-contrib.d/vmware.conf

```diff
@@ -421,6 +421,7 @@ object CheckCommand "vmware-esx-soap-host-net" {
 		"--isregexp" = {
 			set_if = "$vmware_isregexp$"
 		}
+		"--unplugged_nics_state" = "$vmware_unplugged_nics_state$"
```

Comment:
Could you please set this value to a [CheckCommand Argument object](https://icinga.com/docs/icinga-2/latest/doc/09-object-types/#checkcommand-arguments) including both a `description` like the one you have provided for the docs and set `value` to the variable? I am aware that this is not something done stringently throughout this file, but having a description may help for debugging or a future refactoring.

Same applies for the other change in `vmware-esx-soap-host-net-nic`.

Reply:
done